helpers/presentation_objects.py

```diff
@@ -224,7 +224,7 @@ def load(self, dst_folder, **kwargs):
             plt.close(fig)
 
         frames = [imageio.imread(png) for png in sorted(png_dir.iterdir())]
-        imageio.imwrite(dst_folder / gif_file, frames, duration=1 / 2000, loop=0)
+        imageio.imwrite(dst_folder / gif_file, frames, duration=5e2, loop=0)
```

Comment:
Why `5e2` and not `500`?

Reply:
because I also tried 1000 and there it's just a habit for me to write 1e3, haha. But I was wondering if the current version is too fast, see the GIF in my other comment. what do you think?

Reply:
I think it looks okay, but the perception may of course change from person to person and also for different variables. I would say we go for it. But I would definitely prefer 500 for the sake of readability (and to not disclose the Fortran heritage :grin:)